scripts/migrate-google-sso-users.rb

```diff
@@ -0,0 +1,97 @@
+require 'csv'
+require 'uri'
+require File.expand_path("../lib/uaa_sync_admin_users", __FILE__)
+
+module CF
+  module UAA
+    module Http
+      def json_parse_reply(style, status, body, headers)
+        raise ArgumentError unless style.nil? || style.is_a?(Symbol)
+        unless [200, 201, 204, 400, 401, 403, 409, 422].include? status
+          raise (status == 404 ? NotFound : BadResponse), "invalid status response: #{status}"
+        end
+        if body && !body.empty? && (status == 204 || headers.nil? ||
+              headers['content-type'] !~ /application\/json/i)
+          raise BadResponse, 'received invalid response content or type'
+        end
+        parsed_reply = Util.json_parse(body, style)
+        if status >= 400
+          puts parsed_reply.inspect
```

Comment:
I have added this, because the library being used doesn't say what error occurred some of the time which made running this script a pain. Assuming this script will be removed once run I think it's not too ugly.